array/README.md

```diff
@@ -564,6 +564,26 @@ Returns only unique elements of the given array.
 ```
 <!-- prettier-ignore-end -->
 
+# uniqueBy
+
+This method is like unique except that it accepts selector which is invoked for each element in array to generate the criterion by which uniqueness is computed.
+
+## Type signature
+
+<!-- prettier-ignore-start -->
+```typescript
+(selector: (x: any) => any) => (xs: any[]) => any[]
+```
+<!-- prettier-ignore-end -->
+
+## Examples
+
+<!-- prettier-ignore-start -->
+```javascript
+uniqueBy(({ x }) => x)([{ x: 1 }, { x: 2 }, { x: 1 }]) // ⇒ [{ x: 1 }, { x: 2 }]
```

Comment:
Consider expanding this with another field to see how duplicates are overriden (right-to-left):
`uniqueBy(({ id }) => id)([{ id: 1, value: 'a' }, { id: 2, value: 'b' }, { id: 1, value: 'c' }]  // ⇒ [{ x: 2, value: 'b' }, { x: 1, value: 'c' }]`